### ISSUE TITLE
[subviews] IWD1 crash when exiting the game

### ISSUE THREAD

**Issue by FrElvire** [gemrb/gemrb#1236]
#### Bug description
When quitting game after some hours, the game crashed. I don't know if it specific to subviews branch but as it's the branch used...

I don't know how to reproduce (first time in some start/stop), but here the information I have :

```
[ResourceManager]: Searching for 'gam_04'...
[ResourceManager]: Found 'gam_04.wav' in 'chitin.key'.
[Python]: pause state None
[GUIScript]: Loading Script Start.
[ResourceManager]: Found 'guiconn.chu' in 'chitin.key'.
[ResourceManager]: Searching for 'start'...
[ResourceManager]: Found 'start.mos' in 'chitin.key'.
[MUSImporter]: Loading /opt/gemrb/game/Icewind Dale/Music/Theme.mus...
[ResourceManager]: Searching for 'THEME/THEMEA'...
[ResourceManager]: Found 'THEME/THEMEA.acm' in 'Music'.
[MUSImporter]: Playing THEME/THEMEA...

Thread 1 "gemrb" received signal SIGSEGV, Segmentation fault.
0x0000555555dc9740 in ?? ()
(gdb) bt
#0  0x0000555555dc9740 in ?? ()
#1  0x00007ffff7d1202f in GemRB::Control::SetValue (this=0x5555580a18f0, 
    val=1222)
    at /home/dupontr/Downloads/gemrb_sub/gemrb/gemrb/core/GUI/Control.cpp:164
#2  0x00007ffff7d373de in GemRB::ScrollView::UpdateScrollbars (
    this=0x55556b127cc8)
    at /home/dupontr/Downloads/gemrb_sub/gemrb/gemrb/core/GUI/ScrollView.cpp:189
#3  0x00007ffff7d37bb4 in GemRB::ScrollView::ScrollTo (this=0x55556b127cc8, 
    newP=..., duration=0)
    at /home/dupontr/Downloads/gemrb_sub/gemrb/gemrb/core/GUI/ScrollView.cpp:333
#4  0x00007ffff7d37a06 in GemRB::ScrollView::ScrollDelta (this=0x55556b127cc8, 
    p=..., duration=0)
    at /home/dupontr/Downloads/gemrb_sub/gemrb/gemrb/core/GUI/ScrollView.cpp:313
#5  0x00007ffff7d379ac in GemRB::ScrollView::ScrollDelta (this=0x55556b127cc8, 
    p=...)
    at /home/dupontr/Downloads/gemrb_sub/gemrb/gemrb/core/GUI/ScrollView.cpp:308
#6  0x00007ffff7d3b3a0 in GemRB::TextArea::TrimHistory (this=0x55556b127b30, 
    lines=12)
    at /home/dupontr/Downloads/gemrb_sub/gemrb/gemrb/core/GUI/TextArea.cpp:428
--Type <RET> for more, q to quit, c to continue without paging--
#7  0x00007ffff7d3b448 in operator() (__closure=0x55556d16ff68)
    at /home/dupontr/Downloads/gemrb_sub/gemrb/gemrb/core/GUI/TextArea.cpp:452
#8  0x00007ffff7d3cacc in std::__invoke_impl<void, GemRB::TextArea::AppendText(const String&)::<lambda()>&>(std::__invoke_other, struct {...} &) (__f=...)
    at /usr/lib/gcc/x86_64-pc-linux-gnu/10.2.0/include/g++-v10/bits/invoke.h:60
#9  0x00007ffff7d3c9bc in std::__invoke_r<void, GemRB::TextArea::AppendText(const String&)::<lambda()>&>(struct {...} &) (__fn=...)
    at /usr/lib/gcc/x86_64-pc-linux-gnu/10.2.0/include/g++-v10/bits/invoke.h:153
#10 0x00007ffff7d3c8a2 in std::_Function_handler<void(), GemRB::TextArea::AppendText(const String&)::<lambda()> >::_M_invoke(const std::_Any_data &) (
    __functor=...)
    at /usr/lib/gcc/x86_64-pc-linux-gnu/10.2.0/include/g++-v10/bits/std_function.h:291
#11 0x00007ffff7d7cc52 in std::function<void ()>::operator()() const (
    this=0x55556d16ff68)
    at /usr/lib/gcc/x86_64-pc-linux-gnu/10.2.0/include/g++-v10/bits/std_function.h:622
#12 0x00007ffff7de844e in GemRB::Timer::Update (this=0x55556d16ff50, 
    time=15317894)
    at /home/dupontr/Downloads/gemrb_sub/gemrb/gemrb/core/Timer.h:66
#13 0x00007ffff7dd7741 in GemRB::Interface::Main (this=0x55555556eb50)
    at /home/dupontr/Downloads/gemrb_sub/gemrb/gemrb/core/Interface.cpp:935
--Type <RET> for more, q to quit, c to continue without paging--
#14 0x0000555555555385 in main (argc=3, argv=0x7fffffffdba8)
    at /home/dupontr/Downloads/gemrb_sub/gemrb/gemrb/GemRB.cpp:73

```

**Comment by lynxlynxlynx** [comment]
This is the same crash as in #1235.

**Comment by FrElvire** [comment]
but no way to consistently reproduce it... (

**Comment by lynxlynxlynx** [comment]
You need at least 100 lines of text in the message window, either through combat, spamming pause or other such thing. We don't support logging redirection there any more.

**Comment by bradallred** [comment]
this should be fixed by cb60d27
